solana/program_v2_cli/src/commands/network/create/test-keypair.json

```diff
@@ -0,0 +1 @@
+[128,183,42,26,46,230,38,10,235,115,142,115,246,25,191,181,182,226,140,91,71,207,0,111,203,149,174,120,27,253,126,160,198,139,21,28,219,71,182,4,114,149,74,232,77,155,247,18,17,240,63,81,225,5,101,252,209,57,162,40,55,40,147,194]
```

Comment:
Should this be here? Rather a fixture in a test ?

Reply:
Shouldn't be here... I'll remove